### ISSUE TITLE
(v14): false-positive "Invalid regular expression: Lone quantifier brackets"

### ISSUE THREAD

**Issue by merlinnot** [sindresorhus/eslint-plugin-unicorn#446]
Hi! I noticed a bug in the latest release with some regular expressions, as this one:

```typescript
/\{\{verificationUrl\}\}/gu
```

It makes ESLint crash (not report an error, actually crash).
